src/components/SceneViewPage/SceneViewPage.tsx

```diff
@@ -140,7 +140,12 @@ export default class SceneViewPage extends React.PureComponent<Props> {
             <Container className="back-container">
               <Back absolute onClick={onBack}></Back>
               {isTemplatesEnabled && (
-                <Button secondary onClick={this.handleExportProject.bind(this)}>
+                <Button
+                  secondary
+                  onClick={this.handleExportProject.bind(this)}
+                  loading={!this.props.currentProject}
```

Comment:
should `currentProject` go here instead of `this.props.currentProject`?

Reply:
currentProject is getting the project using the function `getCurrentProject` which value is different than the current project. from the props